tests/voq/voq_helpers.py

```diff
@@ -179,7 +180,30 @@ def check_bgp_kernel_route(host, asicnum, prefix, ipver, interface, present=True
         logger.info("Route %s is removed from remote neighbor: %s/%s", prefix, host.hostname, str(asicnum))
 
 
+def check_no_routes_from_nexthop(asic, nexthop):
+    if ':' in nexthop:
+        ver = '-6'
+    else:
+        ver = '-4'
+    cmd = "ip {} route show | grep {} | wc ".format(ver, "\"" + nexthop + " \"")
```

Comment:
`grep` can catch more than expected. For example, we are going to find route for `192.168.0.1`, while grep will also catch `192.168.0.10`. Please consider `grep -w`.
And also pls consider `wc -l`.
By the way, why don't use `show ip route`?

Reply:
Have added the requested flags to grep and wc commands.

The reason for using 'ip route show' instead of 'show ip route' is that with 12K routes, the 'show ip route' commands takes much longer time.